.gitignore

```diff
@@ -1,3 +1,4 @@
 IMU/PDR/lib/
 results/PDR/*
 !results/PDR/*.py
+results
```

Comment:
Den her linje kommer til at fjerne linje 3. Så vil mine Python commits blive ændret.

Måske skriv `!results/*/*.py` i stedet for PDR og skriv `results/*` i stedet for linje 2.